src/parsec.rs

```diff
@@ -120,11 +119,12 @@ impl ParsecMap {
         log_ident: &LogIdent,
     ) -> (Option<DirectMessage>, bool) {
         // Increase the size before fetching the parsec to satisfy the borrow checker
-        self.count_size(
-            serialisation::serialised_size(&request),
-            msg_version,
-            log_ident,
-        );
+        let ser_size = if let Ok(size) = bincode::serialized_size(&request) {
+            size
+        } else {
+            return (None, false);
```

Comment:
Just to confirm that this bool flag is to ensure no parsec poll to be undertaken in case of cannot fetch a request's serialized_size .

same to the below response case.

Reply:
Yes, essentially this is a hint that doing parsec_poll will yield no new result since nothing was added to the newest parsec.